src/vm/codeman.cpp

```diff
@@ -985,6 +985,13 @@ PTR_VOID GetUnwindDataBlob(TADDR moduleBase, PTR_RUNTIME_FUNCTION pRuntimeFuncti
 
     return pUnwindInfo;
 
+#elif defined(_TARGET_X86_) && defined(FEATURE_PAL)
+    PTR_UNWIND_INFO pUnwindInfo(dac_cast<PTR_UNWIND_INFO>(moduleBase + RUNTIME_FUNCTION__GetUnwindInfoAddress(pRuntimeFunction)));
+
+    *pSize = ALIGN_UP(sizeof(UNWIND_INFO), sizeof(DWORD));
```

Comment:
This does not look right. There is no alignment when the unwind blob is getting produced, and so there should be no alignment here where it is getting consumed either.

You should see a crashes during GC with R2R images because of this bug.